app/main/views/marketplace.py

```diff
@@ -148,7 +151,7 @@ def list_opportunities(framework_slug):
         next_link_args.setlist("page", api_next_link_args.get("page") or ())
 
     return render_template('search/briefs.html',
-                           framework=framework,
+                           framework=frameworks[-1],
```

Comment:
Can we rely on the last one being the 'first' framework? Obviously your unit tests are fine as we have mocked the API client but can we always be sure what order will come from the API?

When hitting PREVIEW api, the frameworks seem to come back in an order I can't quite make out...

API side at least, this is what is happening
https://github.com/alphagov/digitalmarketplace-api/blob/master/app/main/views/frameworks.py#L22

If we are able to rely on this behaviour, I wonder if it's worth commenting to say so...